sys/i2c/i2c_io.c

```diff
@@ -48,7 +49,15 @@ error_code_t i2cSendTo(uint8_t sAddr, uint8_t *buf, uint16_t numBytes) {
   }
 
   /* Mock the transmit */
-
+  if(sAddr == LM75BD_I2C_ADDR){
+    memcpy(&I2CSentData, buf, sizeof(I2CSentData));
+    if(numBytes>sizeof(I2CSentData)){
+      // To Do
+    }
```

Comment:
Maybe add a check to make sure the reg address (first byte in the buffer) is either the temp reg or config reg?

Reply:
Is there anything we do if it matches with it or if it doesn't match. Do we actually throw an incorrect reg error or should we write to the I2CSentData regardless?

Reply:
Either return an error code (and maybe print an error message during `printConsole`). Or just exit the program with a non-zero error code